### PR TITLE
Make sure read endpoints are activated.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

**Comment by leplatrem** [comment]
r+

File: cliquet/resource/viewset.py [hunk 1]
```diff
@@ -153,7 +153,9 @@ def is_endpoint_enabled(self, endpoint_type, resource_name, method,
         Uses the settings to tell so.
         """
         readonly_enabled = asbool(settings.get('readonly'))
-        if readonly_enabled and method not in self.readonly_methods:
+        readonly_method = method.lower() in [m.lower() for m in
```
Comment (tarekziade) [inline cliquet/resource/viewset.py]: nitpick: shouldn't we normalize the other way ? e.g. the request method is always uppercase and readonly_methods could be normalize on startup

Comment (Natim) [inline cliquet/resource/viewset.py]: Actually the bug was that we though that the method was always uppercase, actually it is more always lowercase.

File: cliquet/resource/viewset.py [hunk 2]
```diff
@@ -153,7 +153,9 @@ def is_endpoint_enabled(self, endpoint_type, resource_name, method,
...
+                                             self.readonly_methods]
+        if readonly_enabled and not readonly_method:
```
Comment (ametaireau) [inline cliquet/resource/viewset.py]: why the "not" here? I would have naively expected it to return False when the current method is readonly.

e.g.

```python
if readonly_enabled and readonly_method:
    return False
```

Comment (ametaireau) [inline cliquet/resource/viewset.py]: Please disregard :-)

**Pull request merged**